web-ui/src/main/resources/catalog/components/common/ows/OWSService.js

```diff
@@ -94,19 +94,87 @@
+          var result = ol_parser.read(data);
+          var ol_layer = result.Capability.Layer;
+
+          // If no CRS property is available, we're probably dealing with an older WMS (e.g. 1.1.1)
+          // WMS versions prior to 1.3.0 are not (fully) supported by OL's WMSCapabilities parser
+          if (!result.Capability.Layer.CRS) {
+            var parser = new X2JS();
+            var jsonData = parser.xml_str2json(data);
+
+            for (var p in jsonData) {
+              // Get root element
+              if (jsonData.hasOwnProperty(p)) {
+                var layer = jsonData[p].Capability.Layer;
+
+                // Copy SRS to Layer.CRS
+                ol_layer.CRS = layer.SRS || layer._SRS;
+
+                // Copy _SRS to BoundingBox.crs for each BoundingBox (if equal array length)
+                if (layer.BoundingBox && layer.BoundingBox.length > 0 &&
+                  layer.BoundingBox.length === ol_layer.BoundingBox.length) {
+
+                  for (var i = 0; i < layer.BoundingBox.length; i++) {
+                    var bbox = layer.BoundingBox[i];
+                    ol_layer.BoundingBox[i].crs = bbox._SRS || bbox.SRS;
+                  }
+                }
+
+                // Create EX_GeographicBoundingBox extent from LatLonBoundingBox properties
+                if (layer.LatLonBoundingBox) {
+                  var minx = layer.LatLonBoundingBox._minx;
+                  var miny = layer.LatLonBoundingBox._miny;
+                  var maxx = layer.LatLonBoundingBox._maxx;
+                  var maxy = layer.LatLonBoundingBox._maxy;
+                  if (angular.isDefined(minx) && angular.isDefined(miny) &&
+                      angular.isDefined(maxx) && angular.isDefined(maxy)) {
+                    ol_layer.EX_GeographicBoundingBox = ol.extent.boundingExtent([
+                      [parseFloat(minx), parseFloat(miny)],
+                      [parseFloat(maxx), parseFloat(maxy)]
+                    ]);
+                  }
+                }
+                break;
+              }
+            }
+          }
+
+          return result;
+        };
+
+        var parseWMSCapabilities = function(data, withGroupLayer, getCapabilitiesUrl) {
+
+          var resolvedUrl = gnUrlUtils.urlResolve(getCapabilitiesUrl);
+          resolvedUrl.port = '';
+          getCapabilitiesUrl = resolvedUrl.href;
+
           if (!cachedGetCapabilitiesUrls.hasOwnProperty(getCapabilitiesUrl)) {
-            var parser = new ol.format.WMSCapabilities();
-            cachedGetCapabilitiesUrls[getCapabilitiesUrl] = parser.read(data);
+            cachedGetCapabilitiesUrls[getCapabilitiesUrl] = getWMSCapabilities(data);
           }
+
           var result = angular.copy(cachedGetCapabilitiesUrls[getCapabilitiesUrl], {});
           var layers = [];
           var url = result.Capability.Request.GetMap.
               DCPType[0].HTTP.Get.OnlineResource;
-
+          var bbox = getLayerBounds(result.Capability.Layer);
```

Comment:
Also inherit the geographic bounding box if the "child" does not have that property.